packages/react-checkbox/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -84,19 +79,9 @@ export type CheckboxProps = Omit<ComponentProps<CheckboxSlots>, 'defaultChecked'
      * Whether the checkbox should be rendered as checked by default.
      */
     defaultChecked?: 'mixed' | boolean;
-
-    /**
-     * Required state of the checkbox.
-     */
-    required?: boolean;
-
-    /**
-     * Disabled
-     */
-    disabled?: boolean;
   };
 
 /**
  * State used in rendering Checkbox
  */
-export type CheckboxState = ComponentState<CheckboxSlots> & CheckboxCommons;
+export type CheckboxState = ComponentState<CheckboxSlots> & Pick<CheckboxProps, 'children'> & CheckboxCommons;
```

Comment:
The children don't go to the primary slot (input), but we still need to put them somewhere. I suppose I could put them on the root slot instead.

Reply:
I'll put children on `root` for now, since the children end up being children of the root slot in `renderCheckbox`. But this'll change again if we end up making it so the label isn't the root.